galaxy_ng/app/api/ui/serializers/execution_environment.py

```diff
@@ -142,8 +142,11 @@ def get_layers(self, obj):
         # use the prefetched blob_list and artifact_list instead of obj.blobs and
         # blob._artifacts to cut down on queries made.
         for blob in obj.blob_list:
-            layers.append({"digest": blob.digest, "size": blob.artifact_list[0].size})
-
+            # Manifest can be empty after deleting an image.
```

Comment:
I just want to confirm with @ipanova and @newswangerd if this is really expected or is there something else we need to delete on the deletion endpoint?

Reply:
PR looks good. Want to get @ipanova's feedback here

Reply:
Sorry for the late answer, I might be missing some context, but 'Manifest can be empty after deleting an image' sounds really confusing to me. Manifest per se is an image, so if the image was removed then there should not be any manifest left.

If i understand correctly what you're trying to achieve is to properly display image information after the space was reclaimed but images are still displayed for historical reasons?in that case, there won't be any artifacts left, that's correct, however I'd still display the list of layers, just not showing the size at all or maybe setting it to zero.

With current logic introduced in this PR the displayed manifest that that was 'disk reclaimed' will say that it has 0 blobs which is not true.